src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -360,6 +360,20 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>LatestVersionPlayed__c</fullName>
```

Comment:
Since the other field has an api name of `LastElevateEventPlayed__c` should this fields api name be `LastElevateVersionPlayed__c` for consistency?

Reply:
Works for me!